sui-execution/latest/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -83,11 +84,17 @@ const SUI_AUTHENTICATOR_STATE_CREATE: FunctionIdent = (
     AUTHENTICATOR_STATE_MODULE_NAME,
     ident_str!("create"),
 );
+const SUI_RANDOMNESS_STATE_CREATE: FunctionIdent = (
```

Comment:
This change needs to be coupled with a change in the compiler IIRC -- you may have already done this, but making a note, just in case.

Reply:
Thanks for flagging! Can you point me to the corresponding file that needs to be changed (if it hasn't already been changed in this PR)?

Reply:
I was hoping @tnowacki would swoop in and do that, but my searching has brought me here:

https://github.com/MystenLabs/sui/blob/4c1c5faba302e2227e22d56596990db8f197c332/external-crates/move/crates/move-compiler/src/sui_mode/id_leak.rs#L38-L50

Reply:
I think I already updated that file in this PR, no? Do you see something missing from the update?